### ISSUE TITLE
Black and white cam video on Windows

### ISSUE THREAD

**Issue by Getron** [qTox/qTox#1822]
I have problem with video on my Windows 8.1 x64 machine, if i go to video options, i can just balck and white video see, and this not change at all, if i play with video settings there, nothing happend, and the video still stay black and white.

This change is system wide, for example, after this, i go to skype, and video call someone, it see me just on black-white colors.

I need to go to skype options, and to video, and there is an option in skype, to turn the standard settings of camera (this is a cam settings from system to), and then, it work ok.

But, if i go on qTox on video settings, it change to black and white colors on this step.

My PC cam is Logitech Webcam C930e, and it works perfectly on other softwares. Just qTox turn settings of cam to black and white, system wide, and this is a bug.

Please, fix it, if possible.

**Comment by zetok** [comment]
@Getron if you could please provide more info: https://github.com/tux3/qTox/wiki/Useful-bug-reports

Especially log files.

**Comment by towlie** [comment]
I remember qTox was doing this when it has camera option sliders. Does it happen with latest qTox?

**Comment by Getron** [comment]
@towlie yes, the newest qTox, i have reinstalled today, the newest one.

I have thinked, it is maybe a update problems, and i have reinstaled it completly, with restart of system.

But, the problem stil there.

**Comment by towlie** [comment]
@Getron Can you check the latest qTox version again?

**Comment by Getron** [comment]
@towlie Yes, now is working ok. No problem anymore here. Thanks

**Comment by zetok** [comment]
@Getron if it's fixed, please close the issue

**Comment by Bartimaeus-** [comment]
I have this issue with my webcam, and I'm unable to install the latest Windows versions from the installers (32 and 64 bit). I've included pics of the way video from my webcam shows up and the install error window that I get.

![grayscale blues](https://cloud.githubusercontent.com/assets/2954254/8236431/2923ebc4-159d-11e5-901f-0f0f0b0de2e8.PNG)
![install error](https://cloud.githubusercontent.com/assets/2954254/8236445/4a0ce764-159d-11e5-847e-ffdbe8912ff7.PNG)

**Comment by towlie** [comment]
@Bartimaeus- This just means your downloaded installer is corrupt.
Download it again.

**Comment by ProMcTagonist** [comment]
That's a screenshot of Hangouts... Doesn't that prove your webcam problems are higher-level than qTox has control over?

**Comment by Bartimaeus-** [comment]
This is the same issue as described above: "This change is system wide, for example, after this, i go to skype, and video call someone, it see me just on black-white colors."

My webcam was working perfectly until I did a video call using qTox.

I reinstalled qTox with the latest windows installer and I still have the same issue, although I may not be looking at the correct settings spot. I do not see a place to edit the webcam's settings (I can see the webcam in the video device list and see the preview box in grayscale as before).
